lib/Test/Nginx/Socket.pm

```diff
@@ -3283,6 +3283,20 @@ current test block. Default to "localhost".
 
 Run a piece of Perl code specified as the content of this C<--- init> section before running the tests for the blocks. Note that it is only run once before *all* the repeated requests for this test block.
 
+=head2 post_init
```

Comment:
how about post_setup_server_root

Reply:
LGTM, thanks for advice. :)

The new one makes it easier to understand without a source code background!